scss/_skeletons.scss

```diff
@@ -0,0 +1,61 @@
+// .skeleton-line      - the skeleton container
+// .skeleton-dash      - the "text" replacement
+
+.skeleton-line {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  // We use a &nbsp; to preserve
+  // the font size of the parent
+  &::before {
+    width: 0;
+    content: "\00A0";
+  }
+
+  > .skeleton-dash {
+    width: 100%;
+    height: 1em;
+    background-color: $gray-400;
```

Comment:
Maybe we use `currentColor` or something to allow for easy customization of colors with our text utilities? Wondering if that allows folks to insert these into other examples more readily? I'm not 100% since I've not used these myself, so feel free to discard.

```suggestion
    background-color: currentColor;
```

Reply:
I've updated the color to `currentColor`. I used a grayish color to "decolorize" because to my eyes it was too blackish...

I've tried with different approaches here:

First with `gray-400` 
<img width="921" alt="Screenshot 2020-10-10 at 10 27 18" src="https://user-images.githubusercontent.com/385551/95651196-de1c0200-0ae8-11eb-9925-268f278eee77.png">

New one with `currentColor`
<img width="928" alt="Screenshot 2020-10-10 at 10 28 20" src="https://user-images.githubusercontent.com/385551/95651197-e07e5c00-0ae8-11eb-884e-1c26812c909c.png">

Alternative, use `currentColor + opacity`
<img width="926" alt="Screenshot 2020-10-10 at 10 28 57" src="https://user-images.githubusercontent.com/385551/95651198-e3794c80-0ae8-11eb-82c9-14e84e6ecc17.png">

The best would be to use something like `lighten(currentColor, 50%)` but this is not possible 😅.
Any other idea is welcome, otherwise `currentColor` is a good option. Developers can always use the color utilities so...